libkbfs/keybase_daemon_rpc.go

```diff
@@ -538,6 +549,36 @@ func (k *KeybaseDaemonRPC) processUserPlusKeys(upk keybase1.UserPlusKeys) (
 	return u, nil
 }
 
+// LoadUnverifiedKeys implements the KeybaseDaemon interface for KeybaseDaemonRPC.
+func (k *KeybaseDaemonRPC) LoadUnverifiedKeys(ctx context.Context, uid keybase1.UID) (
+	[]VerifyingKey, []CryptPublicKey, error) {
+	cachedUserInfo := k.getCachedUserInfo(uid)
+	keys := len(cachedUserInfo.UnverifiedVerifyingKeys) +
+		len(cachedUserInfo.UnverifiedCryptPublicKeys)
+	if keys != 0 {
```

Comment:
Hmm. I guess I was thinking if we're checking to see if they owned the key that signed a metadata block then they must've had a valid key at some point -- or the mdserver is hacked or broken. So this should be 0 only in the uncached case but I may as well be explicit about it.